include/z64actor.h

```diff
@@ -254,6 +254,10 @@ if neither of the above are set : blue
 0x2000 : translucent, else opaque
 */
 
+#define DYNA_TRANSFORM_NONE 0 // Actors on top are unaffected by movements of the dynapoly actor.
```

Comment:
Just my opinion but I'd say we should use 0 instead of a define for this, like we usually do with flags.
I think it's particularly misleading for this since most `DynaPolyActor_Init` calls make the argument look like an enum, and it's not clear at first glance that it's actually a set of flags you can combine.

Reply:
It's a bit of an unusual case though. Even though they are treated as individual flags in the code that processes the flags, values 0, 1, and 3 are arguably the only correct states to pass into the function.

Reply:
I'm torn because it really should be 0, it's a flag value so 0 indeed indicates "nothing special"

But having the define is a lot more readable, though it doesn't really matter in the end and you can find out easily what "0" is, it's a lot more convenient to know the argument is about dynapoly transform flags at all

I'll be removing the define and changing its uses to 0, can revert if more people think the define for 0 is good though